src/realm/transaction.cpp

```diff
@@ -24,6 +24,8 @@
 #include <realm/table_view.hpp>
 #include <realm/group_writer.hpp>
 
+#include <iostream>
```

Comment:
Probably this is something not needed, maybe `std::cout` was used for debugging?

Reply:
Yes